thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/template.hbs

```diff
@@ -1,55 +1,307 @@
+      {{#each timeRangeOptions as |range|}}
+        <li class="te-alert-page-hzlist__item {{if range.isActive "te-alert-page-hzlist__item--active"}}" {{action "onRangeOptionClick" range}}>
+          {{range.name}}
+          {{#if (eq range.name "Custom")}}
+            : {{date-range-picker
+              class="te-pill-selectors__range-picker"
+              timePicker=false
+              timePicker24Hour=true
+              timePickerIncrement=timePickerIncrement
+              maxDate=today
+              start=viewRegionStart
+              end=viewRegionEnd
+              ranges=predefinedRanges
+              showCustomRangeLabel=false
+              format=uiDateFormat
+              serverFormat="YYYY-MM-DD HH:mm"
+              applyAction=(action "onRangeSelection")
+            }}
+          {{/if}}
+        </li>
+      {{/each}}
+    </ul>
+  </div>
+
   <div class="te-horizontal-cards te-content-block">
-    <h4 class="te-alert-page__subtitle">Alert Performance</h4>
-    <a class="te-pill-selectors__side-link" {{action "onClickTuneSensitivity" this}}>Customize sensitivity</a>
+    <h4 class="te-alert-page__subtitle">{{if isTunePreviewActive "Compare" "Current"}} Alert Performance</h4>
     <div class="te-horizontal-cards__container">
+
       {{!-- Alert anomaly stats cards --}}
       {{#each anomalyStats as |card|}}
         <ul class="te-horizontal-cards__card">
-          <li class="te-horizontal-cards__card-title">{{card.title}}</li>
-          <li class="te-horizontal-cards__card-value">{{card.value}}
-            <div class="te-horizontal-cards__card-tooltip">
-              <span>
-                <i class="glyphicon glyphicon-question-sign"></i>
-                {{#tooltip-on-element}}{{card.text}}{{/tooltip-on-element}}
-              </span>
-            </div>
+
+          <li class="te-horizontal-cards">
+            <div class="te-horizontal-cards__card-title">{{card.title}}</div>
+            {{#if card.tooltip}}
+              <div class="te-horizontal-cards__card-tooltip">
+                <span>
+                  <i class="glyphicon glyphicon-question-sign"></i>
+                  {{#tooltip-on-element}}{{card.text}}{{/tooltip-on-element}}
+                </span>
+              </div>
+            {{else}}
+              <div class="te-horizontal-cards__card-text">{{card.text}}</div>
+            {{/if}}
           </li>
-          <li class="te-horizontal-cards__card-text">
-            {{#if card.projected}}
-              Projected: <strong>{{card.projected}}</strong>
+
+          <li class="te-horizontal-cards__card-value">
+            <div class="te-horizontal-cards__val-group">
+              {{#if isTunePreviewActive}}
+                <div class="te-horizontal-cards__card-subt">Current</div>
+              {{/if}}
+              <div class="te-horizontal-cards__card-number">
+                {{card.value}}
+                {{#if (and card.units (not-eq card.value "N/A"))}}
```

Comment:
Let's extract this logic to be computed by the component and reference it as a variable in the template because it's not too readable here. It's also good practice to make the template as easy to understand as possible. This logic is also repeated in other places, so it removes repeated logic.

Reply:
+1 to this comment